css/index.css

```diff
@@ -0,0 +1,247 @@
+html {
+    box-sizing: border-box;
+    margin: 0;
+    padding: 0;
+  }
+  *, *:before, *:after {
+    box-sizing: inherit;
+  }
+  body {
+    background: #f8f8f8;
+    color: #303438;
+    font-family: "Lato","Helvetica Neue",Helvetica,Arial,sans-serif;
+    margin: 0;
+    padding: 0;
+  }
+  svg {
+    fill: currentColor;
+  }
+  
+  a {
+    color: #1F0C7B;
+    text-decoration: none;
+  }
+    a:hover {
+      text-decoration: underline;
+      cursor: pointer;
+    }
+  
+  h1 {
+    text-align: center;
+    margin-bottom: 1.5em;
+    font-size: 2.5em;
+    font-family: "Merriweather",serif;
+  }
+  div + h1 {
+    margin-top: 3em;
+  }
+  @media screen and (max-width: 600px) {
+    h1 {
+      font-size: 2.1em;
+    }
+  }
+  
+  .header {
+    width: 100%;
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    background-color: black;
+    color: #f8f8f8;
+  }
+    .header > a {
+      margin: .8em;
+      padding: 0.7em;
```

Comment:
[dust] is `em` the right unit here? .7 of 16px is 11.2px - would you want to keep these as whole-number reasonable increments (12px for example). Although em is relative to the parent so if the header has a different base size I might be wrong about that calculation.

It also opens a larger convo about using em for spacing. USWDS does do this, but my understanding for accessibility practices was that you want rem for font-sizing so that a user can adjust the base font in their device. But that doesn't necessarily mean that we want whitespace to scale also. Like, if their base size is doubled to 32px, we don't necessary want the whitespace to double in size also.

Anyways, worth checking if this page actually looks how you want it to if you adjust the base font size in your browser to simulate that for a user. And also thinking about consistent spacing units instead of 11.2px.

Reply:
Good callout, will make sure this is covered